client/db/interface.go

```diff
@@ -30,6 +30,12 @@ type DB interface {
 	CreateAccount(ai *AccountInfo) error
 	// DisableAccount sets the AccountInfo disabled status to true.
 	DisableAccount(ai *AccountInfo) error
+	// UpdateAccount updates account's info blob.
+	//
+	// The only piece of account info that is likely to change is the encrypted
+	// representation of the private key since changing the key pair itself
+	// effectively creates a new account.
```

Comment:
I suggested some kind of comment here because the updater is overly broad, updating all the fields, but it actually is _incorrect_ if the key pair itself changes because that means it's a new account by definition.  I think some kind of comment to that effect is warranted until/unless a targeted updater is created.

Reply:
Gotcha. Ignore me, @amassarwi

Reply:
Could reword it so it does not say "only piece of account info that is likely to change", because now that I think of it I suppose host or cert etc. could change too.  Just wanted to doc that it's almost certainly incorrect to change any of the keys.

Reply:
Perfect

Reply:
What about ?

```
// It almost certainly incorrect to change any of the keys since changing the key pair itself
// effectively creates a new account.
```

Reply:
That reads well.